olive/evaluator/metric.py

```diff
@@ -202,3 +203,11 @@ def get_latency_config_from_metric(metric: Metric):
             sleep_num = sub_type.metric_config.sleep_num
             break
     return warmup_num, repeat_test_num, sleep_num
+
+
+def localize_metrics_user_config(metrics: List[Metric]) -> List[Metric]:
```

Comment:
>localize [](http://example.com/codeflow?start=4&length=8)

use another term. 

Basically, localize refer to make english to chinese, which is confusing when user refer to globalization/localization.

Reply:
Remove it as we already have the cache.